rewrite-gradle/src/main/java/org/openrewrite/gradle/AddGradleWrapper.java

```diff
@@ -77,7 +79,7 @@ public String getDescription() {
 
     @Override
     public Validated validate(ExecutionContext ctx) {
-        return super.validate(ctx).and(GradleWrapper.validate(ctx, version, distribution, gradleWrapper));
+        return super.validate(ctx).and(GradleWrapper.validate(ctx, version == null ? "latest.release" : version, distribution, gradleWrapper));
```

Comment:
Thinking about this again, it would probably be better to use `org.openrewrite.internal.StringUtils.isBlank(version)` here to ensure that it properly handles null or empty strings.

Reply:
Thank you. I addressed that with 23a9ce7641f3efa35e9c68daab6c6001eafa18ac